runtime/fundamentals/web_standards.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Web Standards and Node APIs"
+---
+
+Deno aims to bridge the gap between the browser and the backend with a set of
+web standard APIs that are familiar to web developers. These APIs are designed
+to be compatible with the web platform, making it easier to write code that can
+run in both the browser and Deno.
+
+Existing Node.js ecosystem compatibility is provided through a set of Node.js
+compatible APIs and Deno namespace APIs. These APIs are designed to be
```

Comment:
Deno namespace APIs don't contribute to Node ecosystem compatibility

Reply:
oh yeah, sorry leftover from an attempt to merge that and the paragraph below, removed.